lucene-jbossmodules-integrationtests/pom.xml

```diff
@@ -86,6 +86,25 @@
             <artifactId>lucene-analyzers-common</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Are these dependencies necessary, even if we already declared them in the server provisioning configuration? The "pom" one, in particular? This is not mentioned in the README...

Reply:
They are indeed necessary. They not declared in the provisioning configuration.. they happen to be in the pom file but that's ignored - we need them in the pom to compile the tests.

Also consider you can override versions at provisioning time, so one needs to be explicit about these.

Reply:
I just noticed it was not really obvious from my comment, but I was talking about the dependency to `org.hibernate.lucene-jbossmodules:lucene-jbossmodules` (both the pom and the zip).

So if I understand correctly:

1. the "pom" dependency is only there so that we have (transitive) dependencies to Lucene and tests can compile;
2. the "zip" dependency is there so that the provisioning plugin will find the zip, because the provisioning plugin will not fetch dependencies by itself.

I guess item 1 doesn't require an explicit warning, as people working with WildFly are used to having "provided" dependencies to the JARs they need. Though if *this* is the reason why you changed the Lucene dependencies from "provided" to "compile", then maybe it's worth mentioning in the README that users can simply do what you did instead of declaring each single Lucene dependency in their own pom.

Item 2, on the other hand, is probably worth mentioning in the README? Just a first step "Add a dependency to the feature-pack ZIP your pom" with a snippet of XML would be enough...

Reply:
sorry, I got confused. Understanding you now: removing both dependencies works fine :+1: